org.eclipse.xtend.maven.plugin/src/test/resources/it/pom.xml

```diff
@@ -13,18 +13,17 @@
 	</properties>
 	<dependencyManagement>
 		<dependencies>
-			<dependency>
-				<groupId>com.google.guava</groupId>
-				<artifactId>guava</artifactId>
-				<version>33.3.1-jre</version>
-			</dependency>
 			<dependency>
 				<groupId>org.eclipse.xtext</groupId>
 				<artifactId>xtext-dev-bom</artifactId>
 				<version>${xtextBOMVersion}</version>
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
```

Comment:
do we know this works?

Reply:
@cdietrich I've just noted that dependency is in the dependency management section, so its version would NOT be taken from the dev BOM.

However, that POM is used as the parent POM of the Xtend Maven Plugin IT projects, and none of such projects uses guava as a dependency. So, that is useless and I'm going to remove it and update the PR.